tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -899,12 +903,11 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     def run(self, test_type, tests):
         """Start all managers in the group"""
         self.logger.debug("Using %i processes" % self.size)
-        type_tests = tests[test_type]
-        if not type_tests:
+        if not tests:
```

Comment:
Is this correct? there can be different type of test in tests, right?

Also looks like this is a clean up. I suggest we do that in a different PR, to speed up the review process.